code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -31,6 +31,8 @@
 			adjust_nutrition(-(HUNGER_FACTOR/10))
 			if(m_intent == MOVE_INTENT_RUN)
 				adjust_nutrition(-(HUNGER_FACTOR/10))
+	if(body_position == LYING_DOWN)
```

Comment:
```suggestion
	if(body_position == LYING_DOWN && !moving_from_pull)
```
This shouldn't affect people being dragged around.

Reply:
Why not?

Reply:
Firstly, there's a slowdown for dragging lying mobs not buckled to a roller bed (which should also affect borgs), so it's not as abuseable as you may believe (circuits don't count because they are already somewhat slow and funny emergent gameplay).
Secondly, it'd be nice not to have projectiles fired by anyone get bodyblocked by a non-incapaticated* mob lying on the ground when the intended target is the person dragging him away from the gunman. Yes, you could toggle combat mode but that'd mean giving up the benefits of not being pushed around and would also be a step toward micromanaging combat mode, which is very unergonomic from experience.

Reply:
That situation could exist, but I still think the ability to make someone completely unhittable with ranged weapons is too abusable, and the main idea of this PR was to avoid that sort of thing.
I think the only way to truly determine the solution is through playing the game and the community's response. 
If it turns out to be a problem, or some members have strong opinions, it's a one-line patch.